app/views/people/show.mobile.haml

```diff
@@ -10,7 +10,20 @@
     %span.description
       = @person.diaspora_handle
     - if user_signed_in? && @person != current_user.person
-      = render 'aspect_memberships/aspect_membership_dropdown'
+      - if @presenter.full_hash[:block]
+        .pull-right
+          = link_to t("users.privacy_settings.stop_ignoring"),
+            block_path(@presenter.full_hash[:block][:id]),
+            class: "btn btn-danger",
+            method: :delete
+      - else
+        = render "aspect_memberships/aspect_membership_dropdown"
+        = link_to t("users.privacy_settings.start_ignoring"),
```

Comment:
Try this before this line:
```
.pull-right.clear
   = link_to "",
     blocks_path(block: {person_id: @person.id}),
     data: {confirm: t("are_you_sure")},
     method: :post,
     confirm: true,
     class: "entypo-block"

Reply:
And you'd need to change the translation and replace "are_you_sure" as I proposed in https://github.com/diaspora/diaspora/pull/7884#issuecomment-428485259

Reply:
Change the translation of what?

Reply:
The above didn't work but I was able to get what I think we are looking for.  I'm torn on whether the icon needs to be one size larger or not: 
![image](https://user-images.githubusercontent.com/1683691/46881964-1e254d00-ce1b-11e8-9132-d7ba908bef43.png)

Reply:
Changes to reflect the above have been pushed to the GitHub repo for new review...